lib/fxpay/settings.js

```diff
@@ -81,12 +83,19 @@
   };
 
   exports.configure = function settings_configure(newSettings, opt) {
+    //
+    // Configure new settings values.
+    //
     opt = opt || {};
+
+    // Reset existing configuration.
     if (opt.reset) {
       for (var def in defaultSettings) {
         exports[def] = defaultSettings[def];
       }
     }
+
+    // Merge new values into existing configuration.
     for (var param in newSettings) {
       if (typeof exports[param] === 'undefined') {
```

Comment:
This will always be the case won't it unless `opt.reset` was true? Hence why I'm going down this path trying this out.